mappings/net/minecraft/util/WoodType.mapping

```diff
@@ -0,0 +1,14 @@
+CLASS net/minecraft/class_4719 net/minecraft/util/WoodType
+	FIELD field_21676 OAK Lnet/minecraft/class_4719;
+	FIELD field_21677 SPRUCE Lnet/minecraft/class_4719;
+	FIELD field_21678 BIRCH Lnet/minecraft/class_4719;
+	FIELD field_21679 ACACIA Lnet/minecraft/class_4719;
+	FIELD field_21680 JUNGLE Lnet/minecraft/class_4719;
+	FIELD field_21681 DARK_OAK Lnet/minecraft/class_4719;
+	FIELD field_21682 WOODTYPES Ljava/util/Set;
```

Comment:
`VALUES`?

Reply:
I think `VALUES` is the best. If not, it should at least have an underscore: `WOOD_TYPES`.

Reply:
```suggestion
	FIELD field_21682 VALUES Ljava/util/Set;
```